src/app/exams/exams-add.component.ts

```diff
@@ -114,7 +115,7 @@ export class ExamsAddComponent implements OnInit {
     this.couchService.updateDocument(this.dbName, { createdDate: date, ...examInfo, updatedDate: date }).subscribe((res) => {
       this.documentInfo = { _id: res.id, _rev: res.rev };
       let routerParams = {};
-      if (this.examType === 'courses') {
+      if (this.examType === 'courses' || this.isCourseContent) {
         this.appendExamToCourse(examInfo);
```

Comment:
We should change this to take the `type` route parameter that you're passing so the survey goes on the `survey` property of the course object.  Otherwise we'd overwrite any exams.